apps/backend/src/authn/oidc.strategy.ts

```diff
@@ -36,14 +37,26 @@ export class OidcStrategy extends PassportStrategy(Strategy, 'oidc') {
         userInfoURL: configService.get('OIDC_USER_INFO_URL') || 'disabled',
         clientID: configService.get('OIDC_CLIENTID') || 'disabled',
         clientSecret: configService.get('OIDC_CLIENT_SECRET') || 'disabled',
-        callbackURL: `${configService.get('EXTERNAL_URL')}/authn/oidc/callback`,
-        scope: 'openid profile email'
+        callbackURL: `${configService.get('EXTERNAL_URL')}/authn/oidc_callback`,
+        pkce:
+          configService.get('OIDC_USES_PKCE_S256') === 'true'
+            ? 'S256'
+            : configService.get('OIDC_USES_PKCE_PLAIN') === 'true'
+              ? 'plain'
+              : undefined,
+        scope: ['openid', 'email', 'profile'],
+        skipUserProfile: false,
+        proxy:
+          configService.get('OIDC_USE_HTTPS_PROXY') === 'true'
+            ? true
+            : undefined,
```

Comment:
What is the purpose of setting this to undefined instead of false?

Reply:
Idk just personal preference to only set optional parameters to things when I am using them.  in this case, i'd only be using this parameter if we set it to true so i let it do its default behavior otherwise.